metadata-proxy/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM gcr.io/google_containers/nginx
+MAINTAINER Quintin Lee <qlee@google.com>
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install -y \
+  iptables \
+  sudo
```

Comment:
Ok, I removed sudo. The nginx master process starts as root, and then forks www-data worker processes.